.github/workflows/build.yml

```diff
@@ -1,30 +1,27 @@
-name: Build documentation pages
-
+name: Build Examples
 on:
   push:
-    branches:
-    - '*'
-#  pull_request_target:
-#    branches:
-#    - '*'
-
+    branches-ignore:
+      - main
 jobs:
-  build-and-run:
-    name: Build documentation pages
+  build-examples:
+    if: "!contains(github.event.head_commit.message, 'trigger_yapf')"
+    name: build_examples
     runs-on: ubuntu-latest
-
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
-    - uses: actions/checkout@v2
-    - name: Create conda environment
-      env: 
-        ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
-      uses: goanpeca/setup-miniconda@v1
-      with:
-         activate-environment: geocat-examples
-         environment-file: conda_environment.yml
-    - name: Build sphinx docs
-      shell: bash -l {0}
-      run: |
-        conda info
-        conda list
```

Comment:
Unless there is a specific reason for deletion, I'd suggest keeping `conda info` and `conda list` in the config, maybe in the "conda_env" step, as they have been useful especially when we get errors with dependencies.

Reply:
they were only there for testing, I will add them back as needed if there is are any errors, but I want to keep this to only the instructions that are actually needed, so that it self-documents a little better.